server/api/views.py

```diff
@@ -158,3 +161,24 @@ def get(self, request, *args, **kwargs):
             writer.writerow(row)
 
         return response
+    
+class UploadCSVtoAnalyseView(APIView):
+    def get_serializer(self, queryset, many=True):
+        return self.serializer_class(
+            queryset,
+            many=many,
+        )
+    def get(self, request, *args, **kwargs):
+        export_url = reverse('transactions-export')
```

Comment:
It's a bit inefficient and will perform poorly with large files, because you ask Django to download a file, keep it in memory, then parse that file. Better to use `Transaction.objects` and create a dataframe from that